src/scripts/records/UserRecords.js

```diff
@@ -1,31 +1,31 @@
 /* @flow */
 
-import { Record as ImmutableRecord } from 'immutable'
+import Immutable from 'immutable'
 import Cookies from 'js-cookie'
 
-class User extends ImmutableRecord({
-  email: null,
-  name: 'John Doe', // TODO
-  isLoggingIn: false,
-  keepUrl: true
-}) {
-  email: string;
-  name: string;
-  isLoggingIn: boolean;
```

Comment:
Keeping things in state like `isLogginIn` are error prone, in my opinion. This should be a calculated piece of state (something more fit for a selector). Thus, I have switched to storing `loginRequestStatus`, updating that via actions, and creating an `isLogginIn` selector that reads this new status property